README.md

```diff
@@ -1 +1,150 @@
-# math_magician
\ No newline at end of file
+<a name="readme-top"></a>
+
+  <h3><b> Math Magicians</b></h3>
+
+</div>
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 [ Math Magicians] <a name="about-project"></a>
+"Math magicians" is a website for all fans of mathematics. It is a Single Page App (SPA) that allows users to:
+
+-Make simple calculations.
+-Read a random math-related quote.
+
+
+
+## 🛠 Built With <a name="built-with"></a>
+By building this application, I use React.
+
+### Html/Css/JavaScript <a name="tech-stack"></a>
+React.
+
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+Key Features
+
+key features of the application:
+ -Make simple calculations.
+-Read a random math-related quote.
+
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+- [Live Demo Link]--Coming soon
+
+
+<p align="right">(<a href="#readme-top"></a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Web browser.
+- Code editor eg [Visual Studio Code](https://code.visualstudio.com/download).
+- Github: Generate an ssh key or use https to clone the repository.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+- With a git cmd, git bash or visual studio terminal, clone repo with **``git@github.com:coker2019/math_magician.git``** to download the repository locally on your machine.
+- Then you navigate to the folder in your code editor with the command *``cd coker2019/math-magician``*.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+
+👤 **Maurice Coker**
+
+- GitHub: [@coker2019](https://github.com/coker2019)
+- Twitter: [@Olabisi_coker](https://twitter.com/Olabisi_coker)
+- LinkedIn: [maurice-coker-174b129b](https://linkedin.com/in/maurice-coker-174b129b)
+
+<!-- 👤 **Author2**
+
+- GitHub: [@githubhandle](https://github.com/githubhandle)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle) -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+- send and receive data from API
+- Live Demo.
```

Comment:
- [ ] Kindly make this into checkbox, so you can check it anytime you finish doing the feature, checkbox in markup is done like this `- [ ]`  👇 
- [ ] This is an example of checkbox, note: there should be space after the hyphen and between the square brackets.